### PR TITLE
[improve] adjust the code framework related to CDC.

### PR DESCRIPTION
**Pull request by bingquanzhao**
Adjust the code framework related to CDC.

### CODE REVIEW

File: flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/serializer/jsondebezium/CdcSchemaChange.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+// under the License.
+
+package org.apache.doris.flink.sink.writer.serializer.jsondebezium;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.apache.doris.flink.sink.writer.ChangeEvent;
+
+import java.io.IOException;
+
+public abstract class CdcSchemaChange implements ChangeEvent {
```
Comment (JNSimba) [inline flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/serializer/jsondebezium/CdcSchemaChange.java:25]: Add class annotation to class

File: flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/serializer/jsondebezium/CdcDataChange.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+package org.apache.doris.flink.sink.writer.serializer.jsondebezium;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.apache.doris.flink.sink.writer.ChangeEvent;
+import org.apache.doris.flink.sink.writer.serializer.DorisRecord;
+
+import java.io.IOException;
+import java.util.Map;
+
+public abstract class CdcDataChange implements ChangeEvent {
```
Comment (JNSimba) [inline flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/serializer/jsondebezium/CdcDataChange.java:27]: Does cdcdatachange refer to the debezium type data format?

File: flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/JdbcSourceSchema.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+package org.apache.doris.flink.tools.cdc;
+
+import org.apache.doris.flink.catalog.doris.FieldSchema;
+
+import java.sql.DatabaseMetaData;
+import java.sql.ResultSet;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+
+public abstract class JdbcSourceSchema extends SourceSchema {
```
Comment (JNSimba) [inline flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/JdbcSourceSchema.java:27]: Add class Comment to class

File: flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/serializer/jsondebezium/CdcDataChange.java (hunk 2 above)
Comment (bingquanzhao) [reply flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/serializer/jsondebezium/CdcDataChange.java:31]: > Does cdcdatachange refer to the debezium type data format?

yes

Comment (JNSimba) [reply flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/serializer/jsondebezium/CdcDataChange.java:31]: Is it better to change it to DebeziumDataChange?

**Review by JNSimba — APPROVED** [review]
LGTM

**Comment by JNSimba** [comment]
checkstyle failed, PTAL @bingquanzhao

**Review by JNSimba — APPROVED** [review]
LGTM

**Pull request merged**